### PR TITLE
Warning and hide add attendee for disabled event

### PR DESCRIPTION
**Pull request by ztolley**
* If an event is disabled then hide the add button and show a message explaining why you cannot add.
* Shows any attendees there may be if the event is incomplete as people can add attendees through another route.
* Shows indicator in search results if an event is disabled.
* Show indicator in event details if disabled.

### CODE REVIEW

File: src/apps/events/attendees/views/list.njk [hunk 1]
```diff
@@ -2,17 +2,22 @@
 
 {% block main_grid_right_column %}
   <h2 class="heading-medium">Event Attendees</h2>
-  {% if incompleteEvent %}
+  {% if event.disabled_on %}
+    <p class="c-message c-message--info">
+      You cannot add an event attendee to this page because the event
```
Comment (reupen) [inline src/apps/events/attendees/views/list.njk]: 'to this page' doesn't make much sense – the incomplete message below says 'on this page'.

Comment (ztolley) [reply src/apps/events/attendees/views/list.njk]: Yeah it should be some wording about event, not page.

**Comment by ztolley** [comment]
@yeahfro hmm gonna be a pain to do that, there's no way to mark an event as disabled in the frontend so I'll have to add that in as test data... which highlights the issue that we don't do anything to handle disabled events too.. rats.... ok I may have to do pr's for adding test data and another one for showing the event is disabled in results, detail and disable the edit button...

File: src/apps/events/attendees/views/list.njk (hunk 1 above)
Comment (ztolley) [reply src/apps/events/attendees/views/list.njk]: The incomplete text has previously been signed off. I'll remove it from this one as it doesn't make much sense, though technically you could add one through another route.

**Pull request merged**